components/gcp/gcp.app.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  type: "app",
+  app: "gcp",
```

Comment:
This isn't well-documented - we're planning to add each of the "app name slugs" to this repo so it's more clear - but the slug here should be `google_cloud` to reference the Google Cloud app.

Reply:
Done. I also renamed the containing directory to `google_cloud` as well.